sdcm/cluster.py

```diff
@@ -716,16 +716,21 @@ def start_network_interface(self, interface_name="eth1"):
         self.remoter.sudo(startup_interface_command.format(interface_name))
 
     @cached_property
-    def is_enterprise(self):
+    def is_enterprise(self) -> bool | None:
+        _is_enterprise = None
         if self.distro.is_rhel_like:
-            result = self.remoter.sudo("yum search scylla-enterprise 2>&1", ignore_status=True).stdout
-            if 'One of the configured repositories failed (Extra Packages for Enterprise Linux 7 - x86_64)' in result:
-                return "enterprise" in self.remoter.sudo("cat /etc/yum.repos.d/scylla.repo").stdout
-            return "scylla-enterprise.x86_64" in result or "No matches found" not in result
+            result = self.remoter.sudo("yum search scylla-enterprise 2>&1", ignore_status=True)
+            if result.ok:
+                _is_enterprise = "scylla-enterprise.x86_64" in result.stdout or "No matches found" not in result.stdout
```

Comment:
What about `enterprise` on `Arm`?

Reply:
I guess the other logic would cover it (i.e. No matches)

I just moved it around, the already available logic.